library/xml-model/src/main/java/org/cru/godtools/xml/model/Manifest.kt

```diff
@@ -93,6 +94,11 @@ class Manifest : BaseModel, Styles {
     internal val backgroundImageGravity: ImageGravity
     val backgroundImageScaleType: ImageScaleType
 
+    @ColorInt
+    private val _cardBackgroundColor: Int?
+    @get:ColorInt
+    val cardBackgroundColor: Int get() = _cardBackgroundColor ?: manifest.backgroundColor
```

Comment:
This should be all you need here

```suggestion
    @ColorInt
    private val _cardBackgroundColor: Int?
    @get:ColorInt
    val cardBackgroundColor: Int get() = _cardBackgroundColor ?: backgroundColor
```

Reply:
all `manifest` does on the manifest itself is return `this`